Misc/NEWS.d/next/Library/2018-02-12-05-10-28.bpo-30825.t0RXql.rst

```diff
@@ -0,0 +1 @@
+csv.Sniffer now detects line terminator instead of defaulting to '\r\n'
```

Comment:
Please use **:class:** modification.
```
:class:`csv.Sniffer` now detects line terminator instead of defaulting to '\r\n'.
```

Reply:
@rruuaanng addressed in https://github.com/python/cpython/pull/18336/commits/e1e53168c7fac64d8ee0ab749e68905b2a4d0611